source/_integrations/sensor.rest.markdown

```diff
@@ -133,6 +133,11 @@ force_update:
   required: false
   type: boolean
   default: false
+scan_interval:
```

Comment:
Scan interval is not an property of this integration, it is a property of the platform configuration (not this integration).

Hence not documented here, but here:
<https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval>